### PR TITLE
fix: default value for dates

### PR DESCRIPTION
**Pull request by Ad96el**

### CODE REVIEW

File: frontend/src/api/endpoints.ts
Comment (Ad96el) [inline frontend/src/api/endpoints.ts:1]: I don't know why, but the fmt for this file wrong.

File: frontend/src/components/AttestationAdd.tsx [hunk 1]
```diff
@@ -3,10 +3,22 @@ import TextField from '@mui/material/TextField'
...
 import { useState } from 'react'
 import ReactJson, { InteractionProps } from 'react-json-view'
+
 import { fetchCType } from '../utils/utils'
 
+function getFormattedDate(): string {
+  const date = new Date()
+  const day = date.getDate().toString().padStart(2, '0')
+  const month = (date.getMonth() + 1).toString().padStart(2, '0')
+  const year = date.getFullYear()
```
Comment (Dudleyneedham) [inline frontend/src/components/AttestationAdd.tsx:13]:
```suggestion
  const year = date.getFullYear().toString()
```

File: frontend/src/api/endpoints.ts [hunk 2]
```diff
@@ -1,19 +1,17 @@
-import axios from "axios";
-
+import axios from 'axios'
 
 export async function getEndpoints(): Promise<Array<string>> {
+  const origin = 'https://dena-attester.kilt.io'
```
Comment (Dudleyneedham) [inline frontend/src/api/endpoints.ts]: This is hardcoded. Shoudl it be?

**Review by Dudleyneedham — CHANGES_REQUESTED** [review]
Formatting seems off.

Otherwise some questions.

**Review by Dudleyneedham — APPROVED** [review]
LGTM

**Pull request merged**